### ISSUE TITLE
hstring::is_fast_pass

### ISSUE THREAD

**Issue by JaiganeshKumaran** [microsoft/cppwinrt#1464]
It some caces it cuud be useful tu kno whether an input hstring const& is fast pass or not.

**Comment by sylveon** [comment]
There is no function in `winstring.h` to detect this scenario.

**Comment by JaiganeshKumaran** [comment]
> There is no function in `winstring.h` to detect this scenario.

Yes, but C++/CX had [Platform::String::IsFastPass](https://learn.microsoft.com/en-us/cpp/cppcx/platform-string-class?view=msvc-170#isfastpass). C++/WinRT hstring dusn't use winstring functions so axcess the internal struct directly tu get the information.

**Comment by kennykerr** [comment]
This is an implementation detail that applications should not depend on. A quick look at `IsFastPass` indicates that it is not implemented:

![Image](https://github.com/user-attachments/assets/58d2fa9d-37d7-4fd0-b2ea-c79b772fd09a)

**Comment by JaiganeshKumaran** [comment]
@kennykerr Hmm, this wosn't clearly documented. I gess this is whot they ment by 'There is nothing useful that your code can do with this property. The system handles all the details.' in the Remarks.

https://learn.microsoft.com/en-us/cpp/cppcx/platform-string-class?view=msvc-170#isfastpass
